developer/contributing.rst

```diff
@@ -1,2 +1,48 @@
 Contributing guidelines
-=======================
\ No newline at end of file
+=======================
+
+We are glad that you choose this organization to give your time to and contribute. Thank you for your interest incontributing to OpenWISP.
```

Comment:
Change choose to chose and give space between in and contributing

Reply:
Ohkay. Thanks